packages/wrangler/src/bundle.ts

```diff
@@ -66,6 +67,7 @@ export async function bundleWorker(
 ): Promise<BundleResult> {
 	const {
 		serveAssetsFromWorker,
+		formatErrorsInDev,
```

Comment:
Given that this is a temporary internal thing, then how about we avoid threading it through all the code (especially the React props) and just use `process.env.FORMAT_WRANGLER_ERRORS === "true"` where it is needed below.

Reply:
good idea, lemme remove it

Reply:
Otherwise LGTM!

Reply:
done